CODEOWNERS.txt

```diff
@@ -0,0 +1,45 @@
+# Add your github username to opt-in to automatic review requests for
+# specific sub-packages here:
```

Comment:
```suggestion
# Despite the name of this file, the people listed here DO NOT OWN the code in astropy subpackages,
# the copyright is held by the project as a whole - see the license file for details. 
# 
# However, github currently (2022) does not offer a mechanism on their website to subscribe to
# a fraction of newly open PRs in astropy. Either one chooses the "watch" the entire repository and 
# receives  a large number of emails or one needs to manually check is new PRs for a certain sub-package
# exist. This file is a workaround for that problem:
# All names listed here by sub-packages will be requested for review for any new PR 
# and thus get an email notification.
# In many cases, the names below will be active sub-package maintainers, but anyone interested
# can open a PR to add their github handle to any subpackage they are interested in.


# Add your github username to opt-in to automatic review requests for
# specific sub-packages here:
```

Reply:
I think you have a typo here:

> receives  a large number of emails or one needs to manually check is new PRs for a certain sub-package

I am not entirely sure what you want to say.

Reply:
"check if"? Maybe just shorten the whole thing a bit to
> Either one chooses to "watch" the entire repository and 
> \# thus receive a large number of emails or one needs to manually search for new PRs tagging a certain sub-package.

Reply:
I want to add an explanation of what this file is used for. It's not part of git, but just the gihub interface. Given the name of the file, anyone who find this file, might assume that the people listed there own the code. There are several aspects:
- if thinking of "own" as "legal possession": We have a license and that states the copyright, not this file.
- if thinking of "own" as "is responsible for": We curate astropy.org/team with the sub-package maintainers, but anyone is welcome to open PRs for any sub-package, even if they don't "own" it. That might be obvious to this crowed, but not necessarily to someone coming in new and browsing the repro.
- Since this this file is opt-in it will never by in sync (and likely will get out of sync more as we add and remove sub-package maintainers in the future). On the other hand, I see that people might want to sign up who are not necessarily the sub-package maintainers. I think that's OK and want to write that down so there is no misunderstanding.

Reply:
Typos still need to be addressed.